resources/definitions/ultimaker2.def.json

```diff
@@ -86,6 +86,9 @@
         },
         "machine_acceleration": {
             "default_value": 3000
+        },
+        "support_z_distance": {
+            "value": "0.1"
```

Comment:
Why do you use a support_z_distance of 0.1? Colorado uses a support_bottom_distance = layer_height and support_top_distance = layer_height * 2. 
And there are quite some overwrites of these settings in the quality profiles. Did you forget to clean them or was this intended?

Reply:
It was intended. The UM2 related profiles have this as a default. 
I think I can do some more cleaning and default to layer_height and 2*layer_height. This seems a sensible default for self support.